docs/en/connector-v2/sink/S3File.md

```diff
@@ -383,6 +404,8 @@ For orc file format simple config with `org.apache.hadoop.fs.s3a.SimpleAWSCreden
     access_key = "xxxxxxxxxxxxxxxxx"
     secret_key = "xxxxxxxxxxxxxxxxx"
     file_format_type = "orc"
+    schema_save_mode = "CREATE_SCHEMA_WHEN_NOT_EXIST"
```

Comment:
Please add examples for `Multiple Table Writes`

Reply:
added